go.mod

```diff
@@ -48,7 +48,7 @@ require (
 	github.com/patrickmn/go-cache v2.1.0+incompatible // indirect
 	github.com/rs/cors v1.6.0
 	github.com/shiena/ansicolor v0.0.0-20151119151921-a422bbe96644
-	github.com/sirupsen/logrus v1.2.0
+	github.com/sirupsen/logrus v1.4.0
```

Comment:
Why this update?

Reply:
It just happened when I was clearing some `btcsuite` import that keep showing up every time I make a change that requires my linter to auto import a dependency.
I will drop that change.